azurerm/resource_arm_hd_insight_clusters.go

```diff
@@ -0,0 +1,918 @@
+package azurerm
+
+// Code based on the terraform.provider plugin by Microsoft (R) AutoRest Code Generator.
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/Azure/azure-sdk-for-go/services/hdinsight/mgmt/2015-03-01-preview/hdinsight"
```

Comment:
is there a stable version of this API available? This API is over 3 years old and still in Preview

Reply:
Taking a look at the [go sdk](https://github.com/Azure/azure-sdk-for-go/tree/master/services/hdinsight), this seems to be the only API.